### DISCUSSION TITLE
Durable Functions’ Distributed Tracing for Javascript

### DISCUSSION THREAD

**Discussion by pivendren** [Azure/azure-functions-durable-extension#2160] [Ideas]
Hi all 👋

I came across this post: https://tsuyoshiushio.medium.com/durable-functions-distributed-tracing-71426fe2246f

Are there any plans for this with JS?

**Comment by davidmrdavid** [comment]
Hi @pivendren, thanks for reaching out.
Yes, we have plans to port this to JS, Python, and all of our supported languages. As far as I know, we don't have an ETA at the moment, but we're actively working on this.

**Reply by pivendren** [reply]
Thanks for letting me know. I will keep an eye out and maybe chip in where I can.
